### PR TITLE
Reformat the repository

### PR DESCRIPTION
**Pull request by theopnv**
### Purpose of this PR

Step 1 of the reformat process described there  http://go/srp2core-docs.
The changes are indentations, extra spaces and lines. No change in the code logic/syntax is expected.

Command used to reformat the repo: `perl ~\unity-meta\Tools\Format\format.pl --nobackups --hgroot $(pwd) .`

---
### Testing status
Nightly: https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/graphics%252Freformat-repo/.yamato%252F_abv.yml%2523all_project_ci_nightly_trunk/4272263/job/pipeline

The Nightly has to match today's (from before the refactor): https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/master/.yamato%252F_abv.yml%2523all_project_ci_nightly_trunk/4266983/job/pipeline

### CODE REVIEW

**Comment by theo-pnv** [comment]
@marctem , I ran the formatting job on Yamato, and got these errors when running the reformat tool locally as well:
https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/graphics%252Freformat-repo/.yamato%252F_formatting.yml%2523formatting/4272785/job/(log:Execution). I think it is not blocking, we just have to tell people not to mind these on their PRs. But I think we might want to fix this.
```
[12:53:31.844 INF] com.unity.shadergraph/Editor/Data/Nodes/Input/Gradient/BlackbodyNode.cs:30 Detected non-replaceable tab char in literal string

[12:53:31.854 INF] ERROR: Error formatting file: com.unity.shadergraph/Editor/Data/Nodes/Input/Gradient/BlackbodyNode.cs

[12:53:32.585 INF] com.unity.shadergraph/Editor/Data/Nodes/Artistic/Normal/NormalBlendNode.cs:73 Detected non-replaceable tab char in literal string

[12:53:32.611 INF] ERROR: Error formatting file: com.unity.shadergraph/Editor/Data/Nodes/Artistic/Normal/NormalBlendNode.cs

[12:53:32.814 INF] com.unity.shadergraph/Editor/Data/Nodes/Procedural/Shape/RoundedPolygonNode.cs:28 Detected non-replaceable tab char in literal string

[12:53:32.819 INF] ERROR: Error formatting file: com.unity.shadergraph/Editor/Data/Nodes/Procedural/Shape/RoundedPolygonNode.cs

[12:53:33.305 INF] com.unity.shadergraph/Editor/Data/Nodes/Math/Vector/SphereMaskNode.cs:28 Detected non-replaceable tab char in literal string

[12:53:33.313 INF] ERROR: Error formatting file: com.unity.shadergraph/Editor/Data/Nodes/Math/Vector/SphereMaskNode.cs
```

Note: This link might help https://internaldocs.hq.unity3d.com/unity-meta/details/#what-is-detected-non-replaceable-tab-char-in-literal-string-about

**Review by marctem — APPROVED** [review]
Shader Graph changes look good, sampled many .cs files, and some .hlsl, .uss, and .uxml and all changes look expected.

**Review by sophiaaar — APPROVED** [review]

**Review by julienf-unity — APPROVED** [review]
We should probably take template files into account for the reformatting. They are essentially HLSL code. Extensions is both used by vfx and shadergraph even though they use different macro syntax for the generation

File: com.unity.visualeffectgraph/Editor/GraphView/Blackboard/VFXBlackboardPropertyView.cs [hunk 1]
```diff
@@ -14,18 +14,19 @@
...
             m_NoEnum = noEnum;
         }
-        public override IEnumerable<int>  filteredOutEnumerators {
+
+        public override IEnumerable<int>  filteredOutEnumerators
+        {
             get
             {
-            return m_NoEnum?new int[] {2 }:null;
+                return m_NoEnum ? new int[] {2 } : null;
```
[the remark this answers is not in the document]
Comment (theo-pnv) [reply com.unity.visualeffectgraph/Editor/GraphView/Blackboard/VFXBlackboardPropertyView.cs:29]: Valid points! I will ask to the team in charge of the reformat tool. Good catch for the {2 }

**Pull request merged**